reco_utils/recommender/deeprec/models/graphrec/lightgcn.py

```diff
@@ -0,0 +1,403 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import tensorflow as tf
+import time
+import os
+import sys
+import numpy as np
+import pandas as pd
+from reco_utils.evaluation.python_evaluation import (
+    map_at_k,
+    ndcg_at_k,
+    precision_at_k,
+    recall_at_k,
+)
+from reco_utils.common.python_utils import get_top_k_scored_items
+
+
+class LightGCN(object):
```

Comment:
Give a brief introduction of the model here.  Below is an example:
class XDeepFMModel(BaseModel):
    """xDeepFM model

    J. Lian, X. Zhou, F. Zhang, Z. Chen, X. Xie, G. Sun, "xDeepFM: Combining Explicit 
    and Implicit Feature Interactions for Recommender Systems", in Proceedings of the 
    24th ACM SIGKDD International Conference on Knowledge Discovery & Data Mining, 
    KDD 2018, London, 2018.
    """